mara_db/views.py

```diff
@@ -23,10 +23,21 @@ def navigation_entry():
                 description=f'The schema of the {alias} db',
                 uri_fn=lambda current_db=alias: flask.url_for('mara_db.index_page', db_alias=current_db))
             for alias, db in config.databases().items()
-            if (isinstance(db, dbs.PostgreSQLDB) and not isinstance(db, dbs.RedshiftDB))
-               or isinstance(db, dbs.MysqlDB)  # for now, only show postgres and mysql schemas
+            if supports_extract_schema(db)
         ])
 
+def supports_extract_schema(db: object):
```

Comment:
This should be a singledispatch one defaulting to false and have specific methods supplied for all the current defined ones (to true) so on can contribute new ones.